controllers/managedmcg_controller.go

```diff
@@ -194,6 +199,17 @@ func (r *ManagedMCGReconciler) reconcilePhases() (reconcile.Result, error) {
 		if err := r.reconcileOCSCSV(); err != nil {
 			return ctrl.Result{}, err
 		}
+
+		logger := log.FromContext(ctx)
+		instance := configv1.ClusterVersion{}
+		if err := r.Client.Get(context.TODO(), req.NamespacedName, &instance); err != nil {
+			return ctrl.Result{}, err
+		}
```

Comment:
no need right now...

Reply:
The cluster version that is fetched here used to get the base_path.
About `base_path` usability:
```
basePath is the path to the plugin's assets. The primary asset it the
manifest file called `plugin-manifest.json`, which is a JSON document
that contains metadata about the plugin and the extensions.
```
Let's keep the infra as is and option open.